exploration/default-registry-and-mf1-compatibility.md

```diff
@@ -133,58 +141,56 @@ For more information, see [Working with Timezones](https://w3c.github.io/timezon
 
 #### Options
 
-All date/time functions support the following options, which 
-provide high-level control over date/time formats:
-- `calendar` (default is locale-specific)
-  - valid [Unicode Calendar Identifier](https://cldr-smoke.unicode.org/spec/main/ldml/tr35.html#UnicodeCalendarIdentifier)
-- `numberingSystem` (default is locale-specific)
-   - valid [Unicode Number System Identifier](https://cldr-smoke.unicode.org/spec/main/ldml/tr35.html#UnicodeNumberSystemIdentifier)
-- `timeZone` (default is system default time zone or UTC)
-  - valid identifier per [BCP175](https://www.rfc-editor.org/rfc/rfc6557)
-
-In addition to the above high-level options, a function can use either the appropriate
-_style_ options for that function
+A function can use either the appropriate _style_ options for that function
 or can use a collection of _field options_ (but not both) to control the formatted 
 output.
 
+If both are specified, the _style_ option takes precedence.
```

Comment:
Did this change? I recall earlier that the _field options_ took precedence. In ECMA-402, we throw an exception. Can we do that here?

Reply:
We can emit an error (specifically `Invalid Expression`). That would result in the fallback rendering. 

This changed from the point of view that we didn't specify previously. Error would perhaps be preferable.

Reply:
```suggestion
If both are specified, an _Invalid Expression_ error MUST be emitted
and a _fallback value_ used as the resolved value of the _expression_.
```